Client/Frontend/Brave Notifications/BraveNotificationsPresenter.swift

```diff
@@ -66,14 +67,12 @@ class BraveNotificationsPresenter: UIViewController {
       }
     }
     
-    if parent == nil {
-      presentingController.addChild(self)
-      presentingController.view.addSubview(view)
-      didMove(toParent: presentingController)
-    }
+    guard let window = presentingController.view.window, parent == nil else { return }
```

Comment:
parent will always be nil as you're no longer adding this vc as a child to anything

Reply:
fixed in [71bd7a3](https://github.com/brave/brave-ios/pull/5271/commits/71bd7a352226f53b3c70d13019e0ece5cb2c70bb)